README.md

```diff
@@ -59,6 +59,32 @@ It is recommended that the Workspace is the `cit-web` folder, and not the `cit`
 
 ## Deployment
 
+### Setting up database users and permissions
+
+In production environments, we don't use the default `postgres` user, which has broad permissions on the database.
+
+Create two database users. The `django` user will be used by the Django application, and only has read access for the database. The `migrator` user has write access and is only needed to make changes to the database (i.e. importing data).
+
+```
+CREATE USER django with PASSWORD '[REPLACE_WITH_REAL_PASSWORD]';
+GRANT SELECT ON ALL TABLES IN SCHEMA public TO django;
+ALTER DEFAULT PRIVILEGES FOR USER django GRANT SELECT ON TABLES TO django;
+
+CREATE USER migrator with PASSWORD '[REPLACE_WITH_REAL_PASSWORD]';
+GRANT SELECT, INSERT, UPDATE, DELETE ON ALL TABLES IN SCHEMA public to migrator;
+ALTER DEFAULT PRIVILEGES FOR USER migrator GRANT SELECT, INSERT, UPDATE, DELETE ON TABLES TO migrator;
+ALTER DEFAULT PRIVILEGES FOR USER migrator GRANT USAGE, SELECT, UPDATE ON SEQUENCES TO migrator;
+```
+
+Add the following environment variables to `.env`:
+
```

Comment:
Can we provide a full production env template, prod.env?

Reply:
good idea, will do

Reply:
secrets can be left out ofc :)